junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+import csv
+import os
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from settings.common import ROOT_DIR
+
+from junction.base.constants import ProposalStatus
+from junction.conferences.models import Conference
+from junction.proposals.models import Proposal, ProposalSectionReviewerVoteValue
+from junction.proposals.permissions import is_conference_moderator
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--conference_slug",
+            default=None,
+            help="Enter the conference slug where to export reviewer votes from",
+        )
+
+        parser.add_argument("--user_id", default=None, help="Enter your user id")
+
+    def handle(self, *args, **options):
+
+        try:
+            conference = Conference.objects.get(slug=options.get("conference_slug"))
+            user = User.objects.get(id=options.get("user_id"))
+        except User.DoesNotExist:
+            self.stdout.write("Invalid user")
+            return
```

Comment:
Exception cases should be returned with a Non-Zero Exit Code.

Reply:
handled. though, just curious, what is the advantage?

Reply:
@palnabarun have made the changes, please rereview